client/src/components/Tooltips.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { InfoIcon, IconProps } from "@chakra-ui/icons";
+import {
+  Text,
+  HStack,
+  VStack,
+  Image,
+  Icon,
+  Tooltip,
+  TooltipProps,
+  ComponentWithAs,
+} from "@chakra-ui/react";
+
+interface AdviceProps {
+  text: string;
+  icon?: ComponentWithAs<"svg", IconProps>;
+}
+
+const AdviceTip: React.FC<TooltipProps> = ({ ...props }) => (
+  <Tooltip
+    bg="white"
+    color="gray.600"
+    border="1px"
+    borderColor="gray.400"
+    shadow="xs"
+    rounded="md"
+    hasArrow
+    arrowSize={10}
+    arrowShadowColor="gray" // NOTE: The "gray.400" style colors don't work in this field.
+    ml="5"
+    mr="5"
+    p="3"
+    /*Anything defined prior to {...props} is overriden by anything passed in*/
+    {...props}
+  />
+);
+
+/**
+ * A tooltip generated on an icon hoverover with a skull that states whatever text you want for the player.
+ * @param text The text you want displayed in this tooltip.
+ * @param icon (optional) Defaults to a filled-in ? info icon. Can be any icon within chakra-ui/icon.
+ * @returns A React.FC Tooltip object where the displayed icon generates the tooltip on hoverover.
+ */
+export const AdviceTooltipIcon: React.FC<AdviceProps> = ({ text, icon }) => {
+  const toolTip = [
+    <HStack px={2}>
+      <Image
+        src={"images/itemimages/yorick.gif"}
+        alt={"Yorick, the Skeleton"}
+        boxSize="30px"
+        fit="contain"
+      />
+      <VStack align="stretch" spacing={0.3}>
+        <HStack>
+          <Text bg="gray.200" p="4" rounded="md" fontSize={"12"}>
```

Comment:
same comment as above, also there is no need to wrap a single HStack in a VStack